packages/gestalt/src/Masonry.js

```diff
@@ -409,9 +411,9 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
         role="listitem"
         style={{
           top: 0,
-          left: 0,
-          transform: `translateX(${left}px) translateY(${top}px)`,
-          WebkitTransform: `translateX(${left}px) translateY(${top}px)`,
+          ...(isRtl ? { right: 0 } : { left: 0 }),
```

Comment:
@dangerismycat should we add a visual integration test? That way we can avoid regressions.

Reply:
Eventually yes — we currently have visual diff testing disabled as there have been numerous issues since the Playwright switch and none of us have had the time to look into them (I'll have to address at least some of them in the near future as I get the old Masonry integration tests up and running again before other work on that component).